platform/ios/docs/guides/UserInteractions.md

```diff
@@ -0,0 +1,47 @@
+# User Interactions in the Mapbox iOS SDK
+
+The Mapbox iOS SDK provides a set of built-in gesture recognizers to developers. These interactions can be customized to fit your use case. You see what gesture recognizers are on your `MGLMapView` by accessing the `gestureRecognizers` property on your map.
+
+## Configuring user interaction
+
+There are several properties on a `MGLMapView` that are built-in ways to enable or disable a set of gesture recognizers. Boolean values are set to `true` by default.
+
+- `zoomEnabled` - Allows the user to zoom in and out by pinching two fingers, double tapping, or using "quick zoom". Accepts Boolean values.
+- `scrollEnabled` - Allows the user to scroll by dragging or swiping one finger. Accepts Boolean values.
+- `rotateEnabled` - Allows the user to rotate by moving two fingers in a circular motion. Accepts Boolean values.
+- `pitchEnabled` - Allows the user to tilt the map by vertically dragging two fingers. Accepts Boolean values.
+- `decelerationRate` - Determines the rate of deceleration after the user lifts their finger. You can set the value using the  `MGLMapViewDecelerationRateNormal`, `MGLMapViewDecelerationRateFast`, or `MGLMapViewDecelerationRateImmediate` constants.
+
+## Individual gestures
+
+|Gesture | name | Description | Related Property |
+|:-------:|:---------|----------------| -----------|
+|Pinch    | `handlePinchGesture:` |Zooms in or out on the map's anchor point | `zoomEnabled` |
+|Rotation | `handleRotateGesture:` | Changes the MGLMapView direction based on the user rotating two fingers in a circular motion | `rotateEnabled` |
+|Single Tap |`handleSingleTapGesture:`  | Selects/Deselects the annotation that you tap. | |
+|Double Tap |`handleDoubleTapGesture:` | Zooms in on the map's anchor point | `zoomEnabled` |
+|Two Finger Tap:| `handleTwoFingerTapGesture:` | Zooms out with the map's anchor point centered | `zoomEnabled` |
+|Pan Gesture| `handlePanGesture:`| Scrolls across mapView (_Note: If_ `MGLUserTrackingModeFollow` _is being used, it will be disabled once the user pans_)| `scrollEnabled` |
+|Two Finger Drag | `handleTwoFingerDragGesture:` | Adjusts the pitch of the `MGLMapView` | `pitchEnabled` |
+|"Quick Zoom" |`handleQuickZoomGesture:` |Tap twice. On second tap, hold your finger on the map and pan up to zoom in, or down to zoom out | `zoomEnabled`|
```

Comment:
As above, we shouldn’t use the term “quick zoom” in documentation, especially since it isn’t any quicker than pinching. (If we want to coin a term for this gesture, “one-finger zoom” would be more apt.)